cluster/common.sh

```diff
@@ -963,6 +969,34 @@ function create-certs {
   KUBECFG_KEY_BASE64=$(cat "${CERT_DIR}/pki/private/kubecfg.key" | base64 | tr -d '\r\n')
   KUBEAPISERVER_CERT_BASE64=$(cat "${CERT_DIR}/pki/issued/kube-apiserver.crt" | base64 | tr -d '\r\n')
   KUBEAPISERVER_KEY_BASE64=$(cat "${CERT_DIR}/pki/private/kube-apiserver.key" | base64 | tr -d '\r\n')
+
+  # Setting up an addition directory (beyond pki) as it is the simplest way to
+  # ensure we get a different CA pair to sign the proxy-client certs and which
+  # we can send CA public key to the user-apiserver to validate communication.
+  AGGR_CERT_DIR="${KUBE_TEMP}/easy-rsa-master/aggregator"
+  AGGR_CA_KEY_BASE64=$(cat "${AGGR_CERT_DIR}/pki/private/ca.key" | base64 | tr -d '\r\n')
+  AGGR_CA_CERT_BASE64=$(cat "${AGGR_CERT_DIR}/pki/ca.crt" | base64 | tr -d '\r\n')
+  AGGR_PROXYCLIENT_CERT_BASE64=$(cat "${AGGR_CERT_DIR}/pki/issued/proxy-client.crt" | base64 | tr -d '\r\n')
+  AGGR_PROXYCLIENT_KEY_BASE64=$(cat "${AGGR_CERT_DIR}/pki/private/proxy-client.key" | base64 | tr -d '\r\n')
+}
+
+function setup-easyrsa {
```

Comment:
I'm surprised this is necessary-- how are other certs getting created at cluster startup time?